### PR TITLE
Ethics Book: Self Reflection Chapter

### PR DESCRIPTION
**Pull request by ninadicara**
### Summary

This PR addresses #1919 and adds a chapter to the ethics book about self-reflection that covers what self-reflection is, what it means to practice self reflection, why it matters, some suggestions of how to do it, and further reading on this topic. 

### List of changes proposed in this PR (pull-request)

Adds a self-reflection chapter which includes the following 
- Intro to what self-reflection is and why it matters (chapter title page)
- A sub-chapter explaining positionality and identity
- A sub-chapter about power, privilege and intersectionality
- A sub-chapter with prompts for self-reflection
- A sub-chapter with further resources to read up on the topic more

### What should a reviewer concentrate their feedback on?

Would really appreciate feedback on...
- [x] Spelling and grammar (done by @aseyq !) 
- [x] The sub-chapters explaining the ideas of reflexivity/positionality and power/privilege - these were tricky to explain concisely so would be good to get views from someone new to the topic and maybe someone who knows it well?
- [x] A review of the prompts - do you think they're useful - are there any missing?
- [x] Check reference links work
- [x] check any other links work - to websites or links to other chapters
- [x] add to glossary?
- [x] Everything looks okay?

### Acknowledging contributors

- [ ] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/main/README.md#contributors) in the README file.
- [X] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/main/README.md#contributors) in the README file:

- [x] @aseyq

## Notes for future
It would eventually be nice to have some case studies, but this is a future feature!

### CODE REVIEW

**Comment by EKaroune** [comment]
@all-contributors please add @aseyq for review

File: book/website/_toc.yml [hunk 1]
```diff
@@ -476,6 +476,17 @@ parts:
...
+    - title: Self-Reflection
+      file: ethical-research/self-reflection
+      sections: 
+      - title: Identity and Positionality
+        file: ethical-research/self-reflection/sr-positionality
+      - title: Power and Privilege
+        file: ethical-research/self-reflection/sr-power
+      - title: Self-Reflection Prompts
+        file: ethical-research/self-reflection/sr-prompts
+      - title: Further Resources
```
Comment (EKaroune) [inline book/website/_toc.yml:488]:
```suggestion
      - title: Resources
```
Just using resources is more consistent with the rest of the book - you can add in all resources in this section.

File: book/website/ethical-research/self-reflection/sr-power.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+(er-self-reflection-power-privilege)= 
+# Power and privilege
+
+When thinking about positionality, it is important to understand how we might hold power and privilege relative to the people we work with, those whose data we are studying, or the problems we are attempting to solve.
```
Comment (EKaroune) [inline book/website/ethical-research/self-reflection/sr-power.md:4]:
```suggestion
When thinking about positionality, it is important to understand how we might hold power and privilege relative to the people we work with, those whose data we are studying, and the problems we are attempting to solve. 
```

File: book/website/ethical-research/self-reflection/sr-power.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
+(er-self-reflection-power-privilege)= 
+# Power and privilege
+
+When thinking about positionality, it is important to understand how we might hold power and privilege relative to the people we work with, those whose data we are studying, or the problems we are attempting to solve. 
+Our experiences of power and privilege or oppression are tightly interwoven with our experiences of class, race, and gender and more {cite:ps}`crenshaw1989demarginalizing, collins1990black`.
```
Comment (EKaroune) [inline book/website/ethical-research/self-reflection/sr-power.md:5]:
```suggestion
Our experiences of power and privilege, or oppression, are tightly interwoven with our experiences of class, race, gender and more {cite:ps}`crenshaw1989demarginalizing, collins1990black`. 
```

File: book/website/ethical-research/self-reflection/sr-power.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
+(er-self-reflection-power-privilege)= 
+# Power and privilege
+
+When thinking about positionality, it is important to understand how we might hold power and privilege relative to the people we work with, those whose data we are studying, or the problems we are attempting to solve. 
+Our experiences of power and privilege or oppression are tightly interwoven with our experiences of class, race, and gender and more {cite:ps}`crenshaw1989demarginalizing, collins1990black`. 
+These are important to consider because themes of power and privilege are present in all of our contexts.
```
Comment (EKaroune) [inline book/website/ethical-research/self-reflection/sr-power.md:6]:
```suggestion
These experiences are important to consider because themes of power and privilege are present in all of our contexts. 
```

File: book/website/ethical-research/self-reflection/sr-power.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+# Power and privilege
+
+When thinking about positionality, it is important to understand how we might hold power and privilege relative to the people we work with, those whose data we are studying, or the problems we are attempting to solve. 
+Our experiences of power and privilege or oppression are tightly interwoven with our experiences of class, race, and gender and more {cite:ps}`crenshaw1989demarginalizing, collins1990black`. 
+These are important to consider because themes of power and privilege are present in all of our contexts. 
+Having privilege can make us unaware of related issues that do not directly affect us.
+
+To illustrate, here are some examples of how power and privilege present as structural issues in data science:  
+<!-- Welcome more examples here! -->
+- Racial injustice and patriarchal systems have meant that data science has historically been dominated by white men. This means that there has not been a diverse group of people to direct the aims of data science work, and so many groups are likely to be overlooked when considering the negative impacts and potential benefits of data science.
```
Comment (EKaroune) [inline book/website/ethical-research/self-reflection/sr-power.md:11]:
```suggestion
- **Racial injustice and patriarchal systems** have meant that data science has historically been dominated by white men. This means that there has not been a diverse group of people to direct the aims of data science work, and so many groups are likely to be overlooked when considering the negative impacts and potential benefits of data science.
```

File: book/website/ethical-research/self-reflection/sr-power.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+- The English language is the default in many programming languages and academic settings, which creates barries to knowledge sharing and inclusion. Rather than English speakers learning other languages, it is generally expected that non-English speakers learn English in order to participate and collaborate, which frames English as the default.
```
Comment (EKaroune) [inline book/website/ethical-research/self-reflection/sr-power.md:12]:
```suggestion
- **The English language is the default** in many programming languages and academic settings, which creates barries to knowledge sharing and inclusion. Rather than English speakers learning other languages, it is generally expected that non-English speakers learn English in order to participate and collaborate, which frames English as the default. 
```

File: book/website/ethical-research/self-reflection/sr-power.md [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+<!-- Welcome more examples here! -->
+- Racial injustice and patriarchal systems have meant that data science has historically been dominated by white men. This means that there has not been a diverse group of people to direct the aims of data science work, and so many groups are likely to be overlooked when considering the negative impacts and potential benefits of data science.
+- The English language is the default in many programming languages and academic settings, which creates barries to knowledge sharing and inclusion. Rather than English speakers learning other languages, it is generally expected that non-English speakers learn English in order to participate and collaborate, which frames English as the default. 
+- Data science work can require huge amounts of energy and computational power, which disproportionality affects the lives of people living in the places most affected by climate change. These countries tend to already be economically disadvantaged relative to others, and have less access to the resources needed to manage the impacts of climate change.
```
Comment (EKaroune) [inline book/website/ethical-research/self-reflection/sr-power.md:13]:
```suggestion
- **Data science work can require huge amounts of energy and computational power**, which disproportionality affects the lives of people living in the places most affected by climate change. These countries tend to already be economically disadvantaged relative to others, and have less access to the resources needed to manage the impacts of climate change. 
```

File: book/website/ethical-research/self-reflection/sr-power.md [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+
+Broader structural aspects of power and privilege can seem too big to counter on our own, but they also play out in individual interactions where we have much more control. 
+By understanding systems and structures of power and oppression, and where we sit within them, we can better identify where change needs to happen, and what we can do to contribute to that change. 
+
+## Intersectionality 
+
+Intersectionality is an important concept to know and understand when thinking about power, privilege and identity. 
+
+The term **intersectionality** refers to how multiple identities can intersect, and so people can be oppressed on the basis of combinations of their identities instead of, or as well as, their separate identities. 
+The term was coined by Kimberlie Crenshaw in 1989 {cite:ps}`crenshaw1989demarginalizing` to describe the way that laws that protected Black people and women did not protect against discrimination that exclusively affected Black women.
```
Comment (EKaroune) [inline book/website/ethical-research/self-reflection/sr-power.md:23]: This sentence does not really make sense to me- maybe you can reword it to make it clearer?

File: book/website/ethical-research/self-reflection/sr-power.md [hunk 9]
```diff
@@ -0,0 +1,27 @@
...
+- Data science work can require huge amounts of energy and computational power, which disproportionality affects the lives of people living in the places most affected by climate change. These countries tend to already be economically disadvantaged relative to others, and have less access to the resources needed to manage the impacts of climate change. 
+
+Broader structural aspects of power and privilege can seem too big to counter on our own, but they also play out in individual interactions where we have much more control. 
+By understanding systems and structures of power and oppression, and where we sit within them, we can better identify where change needs to happen, and what we can do to contribute to that change. 
+
+## Intersectionality 
+
+Intersectionality is an important concept to know and understand when thinking about power, privilege and identity.
```
Comment (EKaroune) [inline book/website/ethical-research/self-reflection/sr-power.md:20]: add a definition to the glossary for this term and link to it.

File: book/website/ethical-research/self-reflection/sr-prompts.md [hunk 10]
```diff
@@ -0,0 +1,50 @@
+(er-self-reflection-promts)=
+# Self Reflection Prompts
+
+Reflexivity is the process of using self-reflection to understand ourselves and our beliefs, and how these influence how we create knowledge through our work in data science. 
+There are lots of different aspects you could consider, and the questions below are some prompts to help get you started. 
+
+## Questions about my own identity 
+
+- What is considered ethical in my culture, that other cultures may see differently?
```
Comment (EKaroune) [inline book/website/ethical-research/self-reflection/sr-prompts.md:9]:
```suggestion
- What is considered ethical in my culture that other cultures may see differently?
```

File: book/website/ethical-research/self-reflection/sr-prompts.md [hunk 11]
```diff
@@ -0,0 +1,50 @@
...
+- What are my own {ref}`Social Graces<er-self-reflection-social-graces>`? 
+- How have my Social Graces shaped my experiences of life? 
+- How have my Social Graces shaped the decisions that have led me to do what I do now?
+- What are the Social Graces of my family and friends? 
+  - Are there Graces where we vary a lot/little?
```
Comment (EKaroune) [inline book/website/ethical-research/self-reflection/sr-prompts.md:14]:
```suggestion
  - Are there Social Graces where we vary a lot/little?
```

File: book/website/ethical-research/self-reflection/sr-prompts.md [hunk 12]
```diff
@@ -0,0 +1,50 @@
...
+
+- What do I believe are 'truths' about the concept I study?
+    - What evidence do I base those truths on? Who developed this evidence?
+- Who do I most commonly cite in my research area, and how are their perspectives similar or different to mine?
+-  Whose perspectives are important in answering this research topic?
+- Am I the best person to answer this question - what might I be missing?
+
+## Questions about my data
+
+- Who isn't included in the data I have collected or am using second-hand?
```
Comment (EKaroune) [inline book/website/ethical-research/self-reflection/sr-prompts.md:45]:
```suggestion
- Who isn't included in the data I have collected or reusing?
```

**Review by EKaroune — APPROVED** [review]
This is looking great Nina. :smile:

You don't have to accept all my comments as they are all just suggestions, but it would be good to add some definitions for the keywords in this chapter and then link them in the text.

File: book/website/ethical-research/self-reflection/sr-positionality.md [hunk 13]
```diff
@@ -0,0 +1,43 @@
+(er-self-reflection-identity-positionality)= 
+# Identity and positionality
+
+Traditional science training may lead us to believe that it is possible to analyse and interpret data truly objectively.
```
[the remark this answers is not in the document]
Comment (malvikasharan) [reply book/website/ethical-research/self-reflection/sr-positionality.md:4]:
```suggestion
Traditional science training may lead us to believe that it is possible to analyse and interpret data objectively.
```

File: book/website/ethical-research/self-reflection.md [hunk 14]
```diff
@@ -0,0 +1,45 @@
...
+
+
+(er-self-reflection-summary)=
+## Summary
+
+Our lived experiences and identities give us a particular lens to view the world through, which can be shaped by where we live, the places we work, the education we receive, or the dominant culture we were brought up in. 
+However, this can also lead us to see a situation or problem in one particular way, or to take approaches that exclude certain people or groups. 
+When it comes to ethical dilemmas, our lived experiences also shape our *personal ethics*, which are the way we make decisions about what is right or wrong. 
+
+Self-reflection is a method we can use to examine our personal assumptions, patterns and actions.
```
[the remark this answers is not in the document]
Comment (malvikasharan) [reply book/website/ethical-research/self-reflection.md:18]:
```suggestion
**Self-reflection** is a method we can use to examine our personal assumptions, patterns and actions. 
```

File: book/website/ethical-research/self-reflection/sr-positionality.md [hunk 15]
```diff
@@ -0,0 +1,43 @@
...
+These could be 'macro' influences like our motivations to address a particular problem, or 'micro' like the way we numerically represent a social concept.  
+While there are lots of benefits to the individual ways we all experience the world, the single view each person takes can also embed unseen biases into how we think and work. 
+
+By better understanding ourselves, we can be more proactive about spotting where we might hold biases or gaps in our knowledge. 
+This means we can be transparent about how our unique view has influenced the way we work, which is beneficial to scientific integrity and reproducibility. 
+It can also help us to identify and work against systemic and structural barriers that might otherwise go unchallenged, in order to make our research environments more inclusive (see {ref}`Activism<er-activism>`).
+
+The placement of our identities, socio-cultural position and epistemological approach to knowledge in the context of our research is called our *positionality*.
```
[the remark this answers is not in the document]
Comment (malvikasharan) [reply book/website/ethical-research/self-reflection/sr-positionality.md:17]: Added!

File: book/website/ethical-research/self-reflection/sr-power.md (hunk 9 above)
Comment (malvikasharan) [reply book/website/ethical-research/self-reflection/sr-power.md:20]: Added

File: book/website/ethical-research/self-reflection/sr-positionality.md (hunk 15 above)
[the remark this answers is not in the document]
Comment (malvikasharan) [reply book/website/ethical-research/self-reflection/sr-positionality.md:17]:
```suggestion
The placement of our identities, socio-cultural position and epistemological approach [{term}`def<Epistimology>`] to knowledge in the context of our research is called our *positionality*.
```

File: book/website/ethical-research/self-reflection/sr-positionality.md [hunk 16]
```diff
@@ -0,0 +1,43 @@
...
+We can understand our positionality through the practice of *reflexivity*, which is the process of using *self-reflection* to better understand how our identities and experiences influence the way we behave in our work.
```
[the remark this answers is not in the document]
Comment (malvikasharan) [reply book/website/ethical-research/self-reflection/sr-positionality.md:18]:
```suggestion
We can understand our positionality through the practice of *reflexivity*, which is the process of using *self-reflection* [{term}`def<Self Reflection>`] to better understand how our identities and experiences influence the way we behave in our work. 
```

File: book/website/ethical-research/self-reflection/sr-positionality.md [hunk 17]
```diff
@@ -0,0 +1,43 @@
...
+
+- **G** = Gender, Geography
+- **R** = Race, Religion
+- **A** = Age, Ability, Appearance
+- **C** = Class, Culture
+- **E** = Ethnicity, Education, Employment
+- **S** = Sexuality, Sexual Orientation, Spirituality
+
+The Graces are a helpful place to start thinking about your intersecting identities.
+It can also be helpful to ask:
```
[the remark this answers is not in the document]
Comment (malvikasharan) [reply book/website/ethical-research/self-reflection/sr-positionality.md:41]:
```suggestion

It can also be helpful to ask:
```

File: book/website/ethical-research/self-reflection/sr-power.md (hunk 2 above)
Comment (malvikasharan) [reply book/website/ethical-research/self-reflection/sr-power.md:4]:
```suggestion
When thinking about positionality [{term}`def<Positionality>`], it is important to understand how we might hold power and privilege relative to the people we work with, those whose data we are studying, or the problems we are attempting to solve. 
```

File: book/website/ethical-research/self-reflection/sr-power.md (hunk 8 above)
Comment (malvikasharan) [reply book/website/ethical-research/self-reflection/sr-power.md:23]:
```suggestion
The term was coined by Kimberlie Crenshaw in 1989 {cite:ps}`crenshaw1989demarginalizing` to describe the way that laws that protected Black people and women separately.
However, Black women were not by default protected against discrimination that exclusively affected Black women. 
```

**Comment by malvikasharan** [comment]
@ninadicara amazing work for all the content, and amazing work with Ali Seyhun in getting the chapter structured.
@EKaroune what a thorough review - thanks a million for that. I have gone through your comments to address them carefully. @ninadicara please go ahead and accept them if you agree with them - and then we are ready to merge the first draft! This is massive Y'all!!! 🙌

**Comment by ninadicara** [comment]
Wow thank you so much for the work on this @EKaroune and @malvikasharan ! You are rock stars! 
I've committed all your suggestions now I think (great idea on the glossary too) and it's looking much stronger. 
I'll merge once I've checked over the preview and resolved the bibliography conflict 😇
